components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/js/apis/{apiId}/index.js

```diff
@@ -518,26 +518,54 @@ $(function () {
                             $("#subscribe-button").html('Subscribe');
                             $("#subscribe-button").removeAttr('disabled');
                             var subscription = jsonData.obj;
-                            var message = "You have successfully subscribed to the API.";
-                            noty({
-                                text: message,
-                                layout: "top",
-                                theme: 'relax',
-                                dismissQueue: true,
-                                type: "alert",
-                                buttons: [
-                                    {addClass: 'btn btn-primary', text: 'View Subscriptions', onClick: function($noty) {
-                                        $noty.close();
-                                        location.href = contextPath + "/applications/" + applicationId + "#subscription";
-                                      }
-                                    },
-                                    {addClass: 'btn btn-default', text: 'Stay on this page', onClick: function($noty) {
-                                        $noty.close();
-                                        location.href = contextPath + "/apis/" + apiId;
-                                      }
-                                    }
-                                  ]
-                            });
+                            var jsonPayload = subscription.workflowResponse.jsonPayload;
+                            
+                            if(jsonPayload != null && jsonPayload != "") {
```

Comment:
what would be the benefit of using this method instead of checking the null value? In my opinion it would be much easier to understand what i was doing within that conditional check when jsonPayload != null  is used (checking for null value.)